test/hotspot/jtreg/serviceability/jvmti/thread/GetAllThreads/allthr01/allthr01.java

```diff
@@ -39,7 +39,7 @@
  * @requires vm.continuations
  * @library /test/lib
  * @compile allthr01.java
- * @run main/othervm/native -Djava.util.concurrent.ForkJoinPool.common.parallelism=1 -agentlib:allthr01 allthr01
+ * @run main/othervm/native -Djdk.virtualThreadScheduler.parallelism=1 -agentlib:allthr01 allthr01
```

Comment:
Some operations temporarily increase the parallelism, I suspect you really want maxPoolSize=1 in these tests.